debian/changelog

```diff
@@ -1,3 +1,12 @@
+kora-util (1.2.0) unstable; urgency=low
+
+  * feature(dynamic): make object_t::at method and rvalues friends.
+    This patch allows to use rvalues as a default parameter for `at`
+    method. It saves from the common mistake, where you want to obtain
+    reference to the object in map if it exists, or default value otherwise.
+
+ -- Evgeny Safronov <division494@gmail.com>  Fri, 02 Oct 2015 12:20:46 +0300
+
```

Comment:
The description of any version should contain the list of whole changes that was made.
Check commits' history.